guides/doc-Managing_Content/master.adoc

```diff
@@ -4,17 +4,14 @@ include::common/header.adoc[]
 
 = {ContentManagementDocTitle}
 
+ifdef::foreman-deb[]
```

Comment:
We previously had a warning for `foreman-el`, but this removes that.

Reply:
At first I though it's maybe not necessary. I've readded it & also simplified the whole "master.adoc" file. Please re-review.